test/e2e/kubectl/kubectl.go

```diff
@@ -2159,3 +2223,71 @@ func startLocalProxy() (srv *httptest.Server, logs *bytes.Buffer) {
 	p.Logger = log.New(logs, "", 0)
 	return httptest.NewServer(p), logs
 }
+
+// createApplyCustomResource asserts that given CustomResource be created and applied
+// without being rejected by client-side validation
+func createApplyCustomResource(resource, namespace, name string, crd *framework.TestCrd) error {
```

Comment:
suggest setting `--cache-dir=<per-createApplyCustomResource-tmp-dir>` to ensure each invocation of createApplyCustomResource() refetches schema and discovery docs

Reply:
actually, openapi schema fetching uses etag verification, so we don't need this